src/pages/dashProvas/modals/newProva.tsx

```diff
@@ -50,8 +50,8 @@ function NewProva({ handleClose, addProva, tipos} : NewProvaProps){
     })
 
     const edicaoOption : FormFieldOption[] = edicaoArray.map(f => ({ label: f, value: f }))
-    edicaoOption.push({ label: 'Reaplicação/PPL/2ªAplicação', value: 'Reaplicação/PPL2'})
-    edicaoOption.push({ label: 'Reaplicação/PPL/3ªAplicação', value: 'Reaplicação/PPL3'})
+    edicaoOption.push({ label: '2ªAplicação', value: 'Reaplicação/PPL2'})
+    edicaoOption.push({ label: '3ªAplicação', value: 'Reaplicação/PPL3'})
```

Comment:
Fiquei confusa 
![image](https://github.com/vcnafacul/client-vcnafacul/assets/86015358/49fc9876-5d8a-466c-9e57-b054781ef444)
O objetivo do card era trocar de Reaplicação para Reaplicação/PPL. Mas também fala sobre o 2ªAplicação e 3ªAplicação. Me pergunto se era só nesse lugar que precisaríamos trocar. Apenas para ter certeza e não mergearmos sem quererm, é isso mesmo? Se for já aprovo aqui

Reply:
Erra só ai mesmo
![image](https://github.com/vcnafacul/client-vcnafacul/assets/46038252/7eed0271-cd20-48c8-bc78-78cd28306bde)

é uma informação explicita de prova apenas